src/test/dummy/1/mapping.json

```diff
@@ -0,0 +1 @@
+{"inputs": {"b":"input_tensor"}, "outputs": {"a": "output_tensor"}}
```

Comment:
Wouldnt it break all the tests? The tests rely on a fact that default dummy has input name `b` and output name `a`